react-native.config.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
   reactNativePath: '.',
   project: {
     ios: {
-      project: './RNTester/RNTester.xcodeproj',
+      project: './RNTester/RNTesterPods.xcworkspace',
```

Comment:
cc @thymikee

Reply:
The config resolution works fine, so this is good to go. You can run from cli using:
```
yarn react-native run-ios --project-path RNTester --scheme RNTester
```
However I can see that the `warnAboutManuallyLinkedLibs` fn we call internally to discover and warn about manually linked projects takes very long in this repo, we might want to introduce a temporary flag to disable it.

Btw, how about using the name `RNTester.xcworkspace` instead of `RNTesterPods` now that it's the only one there? If changed, we could run it with just 

```
yarn react-native run-ios --project-path RNTester
```

Reply:
sounds good!

> Btw, how about using the name RNTester.xcworkspace instead of RNTesterPods now that it's the only one there? If changed, we could run it with just

We could, but let's keep the name as is to avoid confusion, at least until many ppl move over to this new setup.